src/main/config/secor.dev.hr.partition.properties

```diff
@@ -0,0 +1,11 @@
+include=secor.dev.properties
+
+secor.kafka.group=secor_hr_partition
+secor.message.parser.class=com.pinterest.secor.parser.ThriftMessageParser
+
+secor.s3.path=secor_dev/hr_partition
+secor.local.path=/tmp/secor_dev/message_logs/hr_partition
+
+message.timestamp.using.hour=true
```

Comment:
There are already several several sub-classes of TimestampedMessageParser (json, thrift parser e.g.) and they can all benefit from hourly parsing/partitioning.   if I introduce another hourly parser, I would have to duplicate code between those parser classes.

If the timestamped parser understands the daily concept, it would be natural for it to understand hourly concept too.